packages/react/src/components/SuiteHeader/SuiteHeader.story.jsx

```diff
@@ -579,8 +583,70 @@ HeaderWithDataFetching.story = {
   name: 'Header with data fetching',
 };
 
-*/
```

Comment:
The end comment here was accidentally removed, so the new story examples are commented out.

Reply:
@kevinsperrine it was not an accident, I've added some more "unsupported" async stories that use the new auxiliary ways to leverage the `GET /uiresources` API that I've created in this PR. The stories have been commented out on purpose. They make API calls so, if I left them uncommented, the build would fail.

Reply:
@blechner Ah. got it. It's probably better suited to be moved into a README.md file and documented as examples of usage rather than just commented out code in the stories. See the [Table README](https://github.com/carbon-design-system/carbon-addons-iot-react/blob/next/packages/react/src/components/Table/README.md) and stories as an example.

Reply:
@kevinsperrine Thanks for the heads up. I'll create a `README.md` file and move the commented out code there.